Dockerfile.ubuntu

```diff
@@ -345,13 +348,144 @@ COPY MakefileCapi /ovms/
 RUN if [ "$FUZZER_BUILD" == "0" ]; then  debug_bazel_flags=${debug_bazel_flags} make -f MakefileCapi cpp && \
                                          debug_bazel_flags=${debug_bazel_flags} make -f MakefileCapi c; fi ;
 
-ARG ovms_metadata_file
-COPY ${ovms_metadata_file} metadata.json
- 
+
 RUN if [ "$FUZZER_BUILD" == "0" ]; then /ovms/bazel-bin/src/ovms --version && /ovms/bazel-bin/src/ovms; fi;
 
 
 COPY release_files/thirdparty-licenses/ /ovms/release_files/thirdparty-licenses/
 COPY release_files/LICENSE /ovms/release_files/LICENSE
 COPY client /client
 
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ #
+FROM $BUILD_IMAGE as pkg
+
+RUN mkdir /patchelf 
+WORKDIR /patchelf
+# hadolint ignore=DL3003
+RUN wget -q https://github.com/NixOS/patchelf/archive/0.10.tar.gz && \
```

Comment:
Can we use available deb package in version:

patchelf/focal 0.10-2build1 amd64  (ubuntu 20.04)